3.3/examples/django/cleanup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+echo "-> removing openshift.local.* directories..."
+sudo rm -rf openshift.local.*
+
+echo "-> stopping all k8s docker containers..."
+docker ps | awk '{ print $NF " " $1 }' | grep ^k8s_ | awk '{print $2}' |  xargs -l -r docker stop
```

Comment:
`bootstrap.sh` didn't work... that's why it is gone :smile: 
It is complex to bootstrap reliably via scripting, as can be seen in the `hack/test-end-to-end.sh` script.
So, `bootstrap.sh` was gone and the summarized steps went into `demo.md`.

Cleanup is easier, and I found it convenient to have it here. When I am `cd`'ed in the demo's directory, there's no way I'll be calling the cleanup script from `origin/examples/sample-app`...

Maybe this is a sign that a `openshift ex cleanup` command would be useful?